crates/utils/src/request.rs

```diff
@@ -216,17 +216,30 @@ async fn is_image_content_type(client: &Client, test: &Url) -> Result<(), LemmyE
   }
 }
 
+pub fn build_user_agent(settings: &Settings) -> String {
+  format!(
+    "Lemmy/{}; +{}",
+    VERSION,
+    settings.get_protocol_and_hostname()
+  )
+}
```

Comment:
Wouldnt it be better to use just hostname, without protocol?

Reply:
Mast uses the protocol and hostname: https://github.com/allinurl/goaccess/issues/841